cobaya/likelihoods/base_classes/planck_clik.py

```diff
@@ -292,6 +289,7 @@ def execute(command):
     if _clik_verbose:
         process = Popen(command, stdout=PIPE, stderr=STDOUT)
         out = []
+        assert process.stdout
```

Comment:
undhandled expception?

Reply:
Yes, it would have given an exception before too - but stdout should not be none  (the assert just shuts up static checking warnings).